### ISSUE TITLE
Fix taking "Selected region" screenshots from secondary displays

### ISSUE THREAD

**Issue by BloodRaven0** [henrypp/freeshooter#31]
Currently, the "selected region" screenshot (which I have bound to just PrntSc and use as my main method of taking screenshots) only works on the primary display. It doesn't move over to the secondary monitor, and if it's started while the cursor is on the secondary monitor, nothing happens, until you move it over to the primary monitor, where it starts automatically as normal.

Just to clarify, this is an issue only with the "selected region" screenshots, the "Fullscreen" and "Active window" work fine with secondary displays.

**Comment by henrypp** [comment]
yes, i think this is a bug. what os? you are using different dpi for monitors?

**Comment by BloodRaven0** [comment]
@henrypp I'm using Windows 10 Pro x64, the monitors might have SLIGHTLY different DPIs, since they're both full HD (1920x1080), but one of them is 23.6 inch, and the other one (my main one) is 23.8 inch. Here's the specs for them:
https://www.displayspecifications.com/en/comparison/cbdc63510
(if you're asking about the windows scaling feature, they're both set to 100%)

I'm not sure if other screenshot apps work with dual monitor setups (or my specific setup), because I've not tried them (even the windows snipping tool). If you want me to try something with my setup, let me know.

**Comment by DigitalNXZ** [comment]
same problem here 🖐
and I have something to add, if you in the select region tool and didn't want to make an Screenshot  you can't press ESC, till you did a rightclick.

**Comment by BloodRaven0** [comment]
> and I have something to add, if you in the select region tool and didn't want to make an Screenshot you can't press ESC, till you did a rightclick.
This happens regardless if you start taking the screenshot on the primary or secondary display, in other words, you ALWAYS have to right-click before you can cancel with ESC. It's kinda annoying, but I've gotten used to it.
What I'm trying to say is, it's better to open a separate issue for this.

**Comment by henrypp** [comment]
@BloodRaven0 @DigitalNXZ FYI, middle click was do the same as Esc key.

**Comment by BloodRaven0** [comment]
@henrypp Wow, that's actually really helpful. Wish it was documented somewhere, so new users could be aware of that functionality. Personally, I expected it to be bound to rightclick, but I guess there were problems with this kind of implementation.

**Comment by henrypp** [comment]
fixed
